### PR TITLE
Doc (QTM-449) Updates Contrbuting File [Updates the steps to create a new component]

### PR DESCRIPTION
**Pull request by alizeleal**
## Description
https://jirasoftware.catho.com.br/browse/QTM-449

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Documentation

### CODE REVIEW

File: CONTRIBUTING.MD [hunk 1]
```diff
@@ -5,69 +5,148 @@
...
 
 ## Contribution Prerequisites
-- Node;
-- Docker for the visual tests.
 
-We have .nvmrc, so just use [nvm](https://github.com/nvm-sh/nvm) to sets the repo default version of node
-```sh
-nvm use
-```
+- To collaborate with this project, for security reasons, a Token is needed. More info in [confluence of the project](https://confluence.catho.com.br/pages/viewpage.action?pageId=51675150);
```
[the remark this answers is not in the document]
Comment (rapahaeru) [reply CONTRIBUTING.MD:16]: Exactly! I copied text from Catho components and didn't see this line.

**Review by rapahaeru — CHANGES_REQUESTED** [review]
Alize, I think you are not using `commitizen` to push your updates.
Please re-commit using it, so semantic-release can version your updates.

**Review by MarcosViniciusPC — APPROVED** [review]
[x] Documentation

**Review by douglasjr97 — APPROVED** [review]
## Review guide
- [x] Documentation

**Pull request merged**